lib/dash/segment_template.js

```diff
@@ -818,6 +822,57 @@ shaka.dash.TimelineSegmentIndex = class extends shaka.media.SegmentIndex {
       const segmentReplacement = position + this.templateInfo_.startNumber;
       const timeReplacement = this.templateInfo_
           .unscaledPresentationTimeOffset + range.unscaledStart;
+      const timestampOffset = this.periodStart_ -
+        this.templateInfo_.scaledPresentationTimeOffset;
+
+      const partialSegmentRefs = [];
+
+      const partialDuration = (range.end - range.start) / range.partialSegments;
+
+      for (let i = 0; i < range.partialSegments; i++) {
+        const start = range.start + partialDuration * i;
+        const end = range.start + partialDuration * (i + 1);
```

Comment:
I think this would be clearer as `const end = start + partialDuration;`

Reply:
Changed!